schemas/index.js

```diff
@@ -38,4 +39,5 @@ module.exports = [
   hormone_therapy,
   radiation,
```

Comment:
@Buwujiu nice catch; i forgot to add these in this PR as its a new schema! This determines the order it appears in the dictionary, so can you please move immunotherapy to be AFTER radiation and before follow-up so that all the treatment types appear together?

Reply:
will do